packages/graph/hash_graph_clients/typescript/README.md

```diff
@@ -0,0 +1,50 @@
+## @hashintel/hash-graph-client@0.0.0
+
+This generator creates TypeScript/JavaScript client that utilizes [axios](https://github.com/axios/axios). The generated Node module can be used in the following environments:
```

Comment:
```suggestion
This generator creates a TypeScript/JavaScript client that utilizes [axios](https://github.com/axios/axios). The generated Node module can be used in the following environments:
```

Reply:
This readme is autogenerated. We can add it to the ignore and take over the contents, though

Reply:
I'm fine with leaving it autogenerated 🤷  less maintenance

Reply:
Because of markdownlint, I think I have to add the readme to ignore.. Will apply your changes!